_pages/04-data.md

```diff
@@ -9,7 +9,7 @@ breadcrumb: data
 
 # Overview
 
-M-Lab publishes its test data in two forms:
+Data collected by M-Lab hosted tests that use our ETL pipeline are published in two forms:
```

Comment:
collected by, seems a bit funny framing. How about:

> Measurement data from experiments hosted on M-Lab, and processed via the ETL pipeline, are published in two forms:

Reply:
sgtm. will update the text in next commit.